my-app/src/app/core/ui/header/breadcrumbs/breadcrumbs.component.html

```diff
@@ -1,3 +1,9 @@
-<div class="breadcrumbs layout-padding vertical-align--center">
-  Courses
+<div *ngIf="breadcrumbsService.notEmptyBreadcrumbs?.length" class="breadcrumbs layout-padding vertical-align--center">
+  <ng-container *ngFor="let breadcrumb of breadcrumbsService.getLinkBreadCrumbs()">
```

Comment:
I would call `breadcrumbsService.getLinkBreadCrumbs()` inside a component class and assign to a property, as right now Angular would trigger the method call on each CD